certified-connectors/DocuMotor/apiDefinition.swagger.json

```diff
@@ -38,7 +38,13 @@
       "post": {
         "responses": {
           "200": {
-            "description": "ok"
+            "description": "ok",
+            "schema":{
+              "type": "string",
+              "format": "binary"
+            },
+            "x-ms-visibility": "important",
```

Comment:
@sebastianehlers I am afraid that this will throw a 500 error again. but we can try